src/twisted/web/_http2.py

```diff
@@ -143,6 +144,8 @@ def connectionMade(self):
         by the L{twisted.web.http._GenericHTTPChannelProtocol} during upgrade
         to HTTP/2.
         """
+        if ITCPTransport.providedBy(self.transport):
```

Comment:
hooray we definitely don't have to care about super() as I fret about below because we already overrode this method